src/entity-manager/EntityManager.ts

```diff
@@ -554,6 +554,7 @@ export class EntityManager {
         }
 
         if (findOptions.where || metadata.primaryColumns.length > 1) {
+            qb.setFindOptions(findOptions);
```

Comment:
qb.setFindOptions() was called later on in this method, but not in this case. Without it, tests were failing because there were no findOptions.